etc/helm/pachyderm/templates/pachd/config-secret.yaml

```diff
@@ -140,6 +143,15 @@ stringData:
       redirect_uris:
       - {{ include "pachyderm.consoleRedirectURI" . | quote }}
     {{- end }}
+    {{- range .Values.oidc.additionalClients }}
+    - id: {{ .id }}
+      name: {{ .name }}
+      secret: {{ .secret }}
+      redirect_uris:
+      {{- range .redirect_uris }}
+      - {{ . }}
+      {{- end }}
+    {{- end }}
```

Comment:
would recommend just spreading what's in the values.yaml instead of manually duplicating it yourself
```yaml
{{- toYaml .Values.oidc.additionalClients | nindent 4 }}
```

### Proof that this works
Template
```yaml
  oidcClients: |
    - id: sample
      name: sample
      secret: sample
      redirect_uris:
      - sample
    {{- toYaml .Values.additionalClients | nindent 4 }}
```
Values
```yaml
additionalClients:
  - id: hi
    name: sup
    secret: secret
    redirect_uris:
      - url1
      - url2
```
Rendered output
```yaml
  oidcClients: |
    - id: sample
      name: sample
      secret: sample
      redirect_uris:
      - sample
    - id: hi
      name: sup
      redirect_uris:
      - url1
      - url2
      secret: secret
```